static/css/style.css

```diff
@@ -0,0 +1,176 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
+
+* {
+  margin: 0%;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+h1,
+p {
+  font-family: Inter, sans-serif;
+  font-style: normal;
+}
+
+h1 {
+  position: static;
+  font-weight: 800;
+  font-size: 3rem;
+  line-height: 3.75rem;
+  text-align: center;
+  letter-spacing: 0.37px;
+  color: #172b4d;
+}
+
+h1 .highlight { color: #36b37e; }
+
+p {
+  font-weight: normal;
+  font-size: 1.25rem;
+  line-height: 2rem;
+  text-align: center;
+  color: #172b4d;
+}
+
+header {
+  width: 100%;
+  position: fixed;
+  z-index: 500;
+}
+
+.nav-mobile {
+  display: none;
+}
+
+.welcome-text {
+  position: static;
```

Comment:
- [ ] Please remove this. It is not necessary for mobile-version since you are using flexbox